contracts/access/AccessControlled.sol

```diff
@@ -32,55 +29,26 @@ abstract contract AccessControlled {
         _;
     }
 
-    /// @notice Ensures that the caller is a registered IP account.
-    /// @dev Modifier that checks if the msg.sender is a registered IP account.
-    /// modules can use this modifier to check if the caller is a registered IP account.
-    /// so that enforce only registered IP Account can call the functions.
-    modifier onlyIpAccount() {
-        if (!IP_ACCOUNT_REGISTRY.isIpAccount(msg.sender)) {
-            revert Errors.AccessControlled__CallerIsNotIpAccount(msg.sender);
-        }
-        _;
-    }
-
```

Comment:
This modifier is for restricting a function only to be called by IPAccount.  It should NOT be removed. 
As a platform, we should provide such a modifier to the ecosystem module developer.

Reply:
Ok, will add it back since it's for the future usecases of the ecosystem (not used by our core codebase).

Question: what cases would require only callable by IPAccount but not requiring a permission restriction? We don't do that in our codebase which extensively interacts with IPAccounts.